src/accordion/accordion.ts

```diff
@@ -33,9 +33,25 @@ let nextId = 0;
   directives: [NgbCollapse]
 })
 export class NgbPanel {
+  /**
+   *  A flag determining whether the panel is disabled or not.
+   *  When disabled, the panel cannot be toggled.
```

Comment:
You are right. Don't know why but I had tabset on my mind while I was writing this....